src/app/views/rolodex_view.js

```diff
@@ -1,6 +1,83 @@
+import $ from 'jquery';
+import _ from 'underscore';
 import Backbone from 'backbone';
+import ContactView from 'app/views/contact_view';
+import ContactDetailView from 'app/views/contact_detail_view';
 
 const RolodexView = Backbone.View.extend({
+  initialize: function(options){
+    this.input = options.input;
+
+    this.cardList = [];//create empty array variable.
+
+    //loop through collection of models and add contacts to cardList
+    this.model.forEach(function(contact) {
+      this.addContact(contact);
+    }, this);
+
+    this.listenTo(this.model, "update", this.render);
+
+    this.listenTo(this.model, "add", this.addContact);
+
+    this.listenTo(this.model, "remove", this.removeContact);
+
+  },
+
+  render: function() {
+    $('#contact-cards').empty();//empties every time rerendered; elements are bound in contact-vew
+    // $('#contact-cards').empty(); <----things were weird when I ran this. Still not sure why this is unneccessary. Is it because of the listenTo update above?
+    //loops through cardList and appends each card to correct tag in index.html
+    this.cardList.forEach(function(card) {
+      card.render();
+      $('#contact-cards').append(card.$el);
+    }, this);
```

Comment:
should be using el.$('#contact-cards').append, that way you're ensuring that the contact cards Id is within the view (el).